packages/components/form/src/phone_field/phone_field.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  isPossiblePhoneNumber,
+  parsePhoneNumberFromString,
+} from "libphonenumber-js";
+import { useFormContext, Controller, RegisterOptions } from "react-hook-form";
+import snakeCase from "lodash/snakeCase";
+import { useTranslation } from "react-i18next";
+
+import { FormField } from "../form_field";
+import { useFormField } from "../use_form_field";
+import PhoneInput, { PhoneInputProps } from "./phone_input";
+
+interface OwnProps {
+  defaultValue?: string;
+  label?: Maybe<string>;
+  name: string;
+  rules?: Exclude<RegisterOptions, "valueAsNumber" | "valueAsDate">;
+}
+
+export type PhoneFieldProps = OwnProps &
+  Omit<PhoneInputProps, "onChange" | "defaultValue">;
+
+const formattedValue = (value: string): string => {
+  const parsedPhoneNumber = parsePhoneNumberFromString(value);
+  return parsedPhoneNumber?.formatNational() ?? "";
+};
+
+export const PhoneField: React.FC<PhoneFieldProps> = ({
+  defaultValue = "",
+  label: labelProp,
+  name,
+  rules,
+  width = "input",
+  zIndex = "formField",
+  ...props
+}): React.ReactElement => {
+  const { control } = useFormContext();
+  const { t } = useTranslation(["anchorForms", "forms"]);
+  const {
+    error,
+    hint,
+    label: formFieldLabel,
+    placeholder,
+  } = useFormField({
+    defaultPlaceholder:
+      t([
+        "anchorForms:placeholders.phone_number",
+        "forms:placeholders.phone_number",
+      ]) ?? "",
+    name,
+  });
+  const label = labelProp ?? formFieldLabel;
+  const inputId = snakeCase(name);
+
+  const phoneNumberValidation = (phoneNumber: string): string | undefined => {
+    const valid = isPossiblePhoneNumber(phoneNumber, "US");
+
+    if (!valid) {
+      return (
+        t("errors.invalidPhoneNumber", { ns: ["anchorForms", "forms"] }) ||
```

Comment:
If I understand this correctly, i18next will look for translations in both `anchorForms` and `forms` namespaces, where anchorForms is the `anchor-ui` default and `forms` would be the user-defined namespace. I had a bit of a time getting this to work correctly, and had tinkered quite a bit before landing on this solution. I wasn't sure if we wanted to have `anchorForms` as a fallback if the field isn't defined in a `forms` namespace, and this followed the pattern of the placeholder in the `useFormField` hook above..

Reply:
@wmurray if you just return true inside this function will `FieldError` kick in? If so that would be the best way to handle this IMO as it has built in error message fallback set up properly. Take a look!

Reply:
I think translations as a whole are botched for this field because I also just noticed that the placeholder isn't working correctly in Storybook.

Reply:
@dgalarza is it worth exploring using the `useFormScope` hook, or is that essentially going to be restating what we have with the initial translation hook? I did something a little like:

```
const { tNamespace } = useFormScope();
const { t } = useTranslation(["anchorForms", "forms"]);
const {t: invalidT} = useTranslation(tNamespace);

....

if (!valid) {
     return invalidT("errors.invalidPhoneNumber") || undefined;
}
```